app/modules/asset/service.server.ts

```diff
@@ -1143,14 +1144,25 @@ export async function getAllEntriesForCreateAndEdit({
       locationExcludedSelected,
       selectedLocation,
       totalLocations,
-      customFields,
     ] = await Promise.all([
       /** Get the categories */
       db.category.findMany({
-        where: { organizationId, id: { not: categorySelected } },
+        where: {
+          organizationId,
+          id: Array.isArray(categorySelected)
```

Comment:
Just a thought. I think we dont need the check, because we can simply do:
```
id: {
  notIn: [...categorySelected]
}
```
Not should work both in the case where `categorySelected` is string or array of strings.